navigation.py

```diff
@@ -385,6 +624,55 @@ def listPage(page_id):
         addDir(item.attrib['label'], url)
 
     if len(items) > 0:
+        xbmcplugin.addSortMethod(handle=addon_handle, sortMethod=xbmcplugin.SORT_METHOD_NONE)
+        xbmcplugin.addSortMethod(handle=addon_handle, sortMethod=xbmcplugin.SORT_METHOD_LABEL)
         xbmcplugin.endOfDirectory(addon_handle, cacheToDisc=True)         
 
-
+def getAssetDetailsFromCache(asset_id):
+    return assetDetailsCache.cacheFunction(skygo.getAssetDetails, asset_id)
+
+def getTMDBDataFromCache(title, year = None, attempt = 1, content='movie'):
+    return TMDBCache.cacheFunction(getTMDBData, title, year, attempt, content)
+
+def getTMDBData(title, year=None, attempt = 1, content='movie'):
+    rating = None
+    poster_path = None
+    tmdb_id = None
+    splitter = [' - ', ': ', ', ']
+    tmdb_api = base64.b64decode('YTAwYzUzOTU0M2JlMGIwODE4YmMxOTRhN2JkOTVlYTU=') # ApiKey Linkinsoldier
+    # tmdb_api = base64.b64decode('YjM0NDkwYzA1NmYwZGQ5ZTNlYzlhZjIxNjdhNzMxZjQ=') # ApiKey from Sandman's Amazon VOD Addon
```

Comment:
`+ tmdb_api = base64.b64decode('YjM0NDkwYzA1NmYwZGQ5ZTNlYzlhZjIxNjdhNzMxZjQ=') # ApiKey from Sandman's Amazon VOD Addon
 `
please remove this line